src/deepsparse/debug_analysis.py

```diff
@@ -363,17 +363,24 @@ def main():
                 "Please set batch size to 1 and try again"
             )
 
+        if args.input_ids_length > args.sequence_length:
+            raise ValueError(
+                f"input_ids_length: {args.input_ids_length} "
+                f"must be less than sequence_length: {args.sequence_length}"
```

Comment:
It can be less than or equal to, right?

Reply:
No. This would mean that we have an "empty' cache (of length zero), and that will crash the engine I think.